website/blog/2022-06-16-resources-migrating-your-react-native-library-to-the-new-architecture.md

```diff
@@ -0,0 +1,112 @@
+
+## Introduction
+
+In this post we are sharing an update on tools and resources to help React Native library developers to migrate their **Native Modules** and **Native Components** to their **New Architecture** equivalents, **TurboModule** and **Fabric Components**.
+
+React Native users leverage a vast number of open-source libraries for their apps. For a complete and consistent ecosystem, it is necessary that these libraries migrate such that everyone can benefit from the unlocked capabilities and performance improvements of the new architecture.
+
+Here is what we’re working on to support library developers to migrate to the new architecture:
+
+- **Documentation:** We are expanding the [New Architecture guide](https://github.com/facebook/react-native-website/pull/3037) on the website to cover more concepts of the new architectures and how to develop your components.
+- **Example Migrations:** We’ve set up two repositories to demonstrate how to migrate a React Native app to the New Architecture ([RNNewArchitectureApp](https://github.com/react-native-community/RNNewArchitectureApp)) and how to create a **Fabric Component** and a **TurboModule** that work with both architectures ([RNNewArchitectureLibraries](https://github.com/react-native-community/RNNewArchitectureLibraries)).
+- **Support:** Earlier this year, we created a [GitHub Working Group](https://github.com/reactwg/react-native-new-architecture/discussions) dedicated to discussion and questions around the New Architecture.
+
+In this post, we will dig deeper into these resources and explain in more detail how you can use them most efficiently. Finally, we will provide a snapshot of the current migration state for the most used React Native libraries.
+
+### Documentation
+
+In the past 6 months, we’ve added a guide on adopting the New Architecture (https://reactnative.dev/docs/new-architecture-intro) and an architecture deep-dive (https://reactnative.dev/architecture/overview) on Fabric. We plan to expand this to include more guides and documentation around creating TurboModules, understanding CodeGen, and more. We hope to have updates to share by the 0.70 release.
+
+At the moment, the **New Architecture** guide covers how to migrate your app (https://reactnative.dev/docs/new-architecture-app-intro) and your libraries (https://reactnative.dev/docs/new-architecture-library-intro) to support the New Architecture properly.
+
+If you are interested in the evolution of this guide, or have feedback, you can follow along on this pull request (https://github.com/facebook/react-native-website/pull/3037).
+
+### Example Migrations
+
+For developers who may want to follow along in code, we’ve prepared two example repositories.
+
+#### RNNewArchitectureApp
```

Comment:
Generally adding links to the headers is something that we want to avoid, since they will always have the `#` link anyway and it might be confusing for the users.

Reply:
Yup, added to the `This repo` fragment in the first sentence!